From 9bb7df3e225ed484b39ffbe9197e2c5a826900d7 Mon Sep 17 00:00:00 2001
From: Ignacio Casal Quinteiro <qignacio@amazon.com>
Date: Mon, 30 Sep 2019 12:51:08 +0200
Subject: [PATCH] meson: fix assertion when checking if gio_module_dir is
 available

We should not assert that gio module dir starts with prefix
since this is not true for windows builds
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index eac24fd..bd9049a 100644
--- a/meson.build
+++ b/meson.build
@@ -64,7 +64,7 @@ else
                                                   define_variable: ['libdir', join_paths(prefix, libdir)])
 endif
 
-assert(gio_module_dir.startswith(prefix), 'GIO_MODULE_DIR is missing from gio-2.0.pc')
+assert(gio_module_dir != '', 'GIO_MODULE_DIR is missing from gio-2.0.pc')
 
 # *** Checks for LibProxy   ***
 libproxy_dep = dependency('libproxy-1.0', version: '>= 0.3.1', required: get_option('libproxy'))
-- 
2.17.1

